cl/users/templates/profile/alerts.html

```diff
@@ -14,16 +14,31 @@
 
 
 {% block content %}
+  {# Navigation Tabs #}
+  <div class="col-xs-12 v-offset-above-2 v-offset-below-2">
+    <ul class="nav nav-pills mobile-tabs">
+      <li class="{% if page == "search_alerts" %} active {% endif %} medium">
+        <a href="{% url "profile_alerts" %}">
+          <i class="fa fa-search"></i> Search Alerts</a>
+      </li>
+      <li class="{% if page == "docket_alerts" %} active {% endif %} medium">
+        <a href="{% url "profile_docket_alerts" %}">
+          <i class="fa fa-file-text-o"></i> Docket Alerts</a>
+      </li>
+    </ul>
+  </div>
```

Comment:
I thought that maybe it wasn't worth it to put it into an include because this is not reused in another template. But thinking about it better for maintainability it's worth it since now we know that all the sub-tabs are into an include, so I added also this into an include.